docs/quick-starts/rug-typescript.md

```diff
@@ -89,51 +89,46 @@ Known operations are []
 With our test *happily* failing you can now write the following Rug TypeScript editor to meet the assertions of the test. Create a file called `SimpleSampleEditor.ts` in the `.atomist/editors` directory that contains the following:
 
 ```typescript
-import { ProjectEditor } from "@atomist/rug/operations/ProjectEditor"
-import { Status, Result, Parameter } from "@atomist/rug/operations/RugOperation"
-import { Project, Pair, File } from '@atomist/rug/model/Core'
-
-let params: Parameter[] = [
-    {
-        name: "description",
-        displayName: "Description",
-        description: "Text to be added to the README.md",
-        validInput: "Any valid text",
-        minLength: 1,
-        pattern: "@any",
-    }
-]
+import { EditProject } from "@atomist/rug/operations/ProjectEditor"
+import { Editor, Tags, Parameter } from "@atomist/rug/operations/Decorators"
+import { Project } from '@atomist/rug/model/Core'
 
-export let editor: ProjectEditor = {
-    tags: ["simple"],
-    name: "SimpleSampleEditor",
-    description: "A simple sample Rug TypeScript editor",
-    parameters: params,
-    edit(project: Project, {description } : {description: string}) {
+@Editor("SimpleSampleEditor","A simple sample Rug TypeScript editor")
+@Tags("simple")
+class MySimpleEditor implements EditProject{
 
-        project.addFile("README.md", description);
+    @Parameter({description: "Does fun things!", pattern: "@any"})
+    description: string = "Hello, Rug TypeScript World!"
 
-        return new Result(Status.Success, "README.md added to project")
+    edit(project: Project) {
+        project.addFile("README.md", this.description);
     }
 }
+
+export let simple = new MySimpleEditor()
 ```
 
 Walking through this editor the contents are:
 
--   Importing the TypeScript types for working with Rug.
--   Declaring a single, mandatory parameter `description` to this editor.
--   Tagging and exporting a `ProjectEditor` implementation that contains the `edit` function implementation for the project-editing logic.
+-   Importing the TypeScript typings for working with Rug.
+-   Declaring the editor using the `@Editor` decorator, passing in its name and description
+-   Tagging the editor using the `@Tag` decorator to aid discoverability
+-   Implementing the `EditProject` interface to enforce the `edit` function signature
+-   Declaring any require parameters using the `@Parameter` decorator
+   - The value of the field is used as the default value if no value is supplied during invocation
```

Comment:
If you want this to be a subitem, it is safest and most portable if you indent four space and have the content start at the eighth character. That said, if there is only one subitem, it is best practice to roll it into the parent.